### ISSUE TITLE
Deployable to Heroku

### ISSUE THREAD

**Issue by konklone** [18F/analytics-reporter#79]
Go through the process of making this deployable on Heroku, so that it can be closer to deployable in any given PaaS environment.

**Comment by rebeccawilliams** [comment]
icymi: https://github.com/codeforamerica/city-analytics-dashboard

**Comment by laurenancona** [comment]
I've deployed to Heroku in the last couple of days, and am happy to document, if needed. http://philagovanalytics.s3.amazonaws.com/ 
Still working on a built-in method to run updates per crontab, considering this: https://www.npmjs.com/package/node-crontab

**Comment by konklone** [comment]
@laurenancona I would drop everything to merge a PR that allowed this app to run entirely in a Heroku environment. A Node-based scheduler (which could be used optionally, like `--publish`) seems like a solid move. 

Check out https://github.com/node-schedule/node-schedule -- it's already pulled into this project's package.json, as part of the code path we're currently _not_ using in production behind analytics.usa.gov. But we'd welcome work to use it for that.

Also, I see that you have a fork going, that you automatically deploy to Heroku. I totally get why you're doing that, but it would also be super neat if we could have this 18F repo be useful to both of our front-ends. This analytics-reporter project is intended to facilitate that, and it'd be nice to be in a position where we're naturally fixing each other's bugs.

**Comment by laurenancona** [comment]
@konklone I should probably have begun with a disclaimer: I'm super familiar with the GA API side of things (was consulting for SaaS companies before joining the Phila.gov data services team a few months ago). I had unknowingly been pursuing a similar rollup implementation for all Phila.gov municipal sites in parallel to this 18F project. You've solved many of the issues with the reporter tool like cache/OAuth that I'd not had time to address - in fact, this was my first time touching all of the backend components: Heroku, S3, etc. Long story.

In short - I'd love to keep working on it for both of our front ends. I need to push some changes - the fork is no longer auto-deploying to Heroku; was just in place because I'm literally so new at this it was easier. That said, I have some (hopefully useful) ideas for contributions I wouldn't mind kicking around

I hadn't seen that node-scheduler - will definitely take a look. We've been talking about a Heroku>gh-pages possibility, too.
